priv/repo/migrations/20180809102052_create_comments.exs

```diff
@@ -0,0 +1,15 @@
+defmodule QbBackend.Repo.Migrations.CreateComments do
+  use Ecto.Migration
+
+  def change do
+    create table(:comments, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :body, :string
+      #add :manual_id, references(:manuals, type: :binary_id)
```

Comment:
You can remove this since we will make a new migration to associate comments to manuals